### PR TITLE
Conceptx Bid Adapter : initial release

### PR DESCRIPTION
**Pull request by cpcpn-emil**
## Type of change
- [x] New bidder adapter  

## Description of change
<!-- For new bidder adapters, please provide the following
- contact email: info@concept.dk
- test parameters for validating bids:
```
{
  bidder: 'conceptx',
  params: {
     site: "example", 
     adunit: "some-id-3"
  }
}
```

### CODE REVIEW

**Comment by cpcpn-emil** [comment]
@spotxslagle, any idea when we can move along with this ?

**Review by spotxslagle — APPROVED** [review]
LGTM. I made some suggestions, but they're code style rather than anything that would be blocking.

File: modules/conceptxBidAdapter.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    const requests = [];
+
+    if (bidderRequest && bidderRequest.gdprConsent && bidderRequest.gdprConsent.gdprApplies) {
+      ENDPOINT_URL += '?gdpr_applies=' + bidderRequest.gdprConsent.gdprApplies;
+      ENDPOINT_URL += '&consentString=' + bidderRequest.gdprConsent.consentString;
+    }
+    for (var i = 0; i < validBidRequests.length; i++) {
+      const requestParent = { adUnits: [], meta: {} };
+      const bid = validBidRequests[i]
+      const { adUnitCode, auctionId, bidId, bidder, bidderRequestId, ortb2 } = bid
```
[the remark this answers is not in the document]
Comment (cpcpn-emil) [reply modules/conceptxBidAdapter.js]: Noted and changed

[the remark this answers is not in the document]
Comment (patmmccann) [reply modules/conceptxBidAdapter.js:27]: @cpcpn-emil auctionid might be null in Prebid 8

**Pull request merged**